files/build_templates/physyncd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=physyncd service
+Requires=database.service updategraph.service
+{% if sonic_asic_platform == 'broadcom' %}
+Requires=opennsl-modules.service
```

Comment:
why the gbsyncd requires opennsl module?

Reply:
I believe because ultimately broadcom-sai has a dependency on it. Looks to me like another placeholder for when we have a hardware based gb.